examples/progress_bar_minimal.py

```diff
@@ -0,0 +1,97 @@
+"""This file provides minimal working examples of progress bars in
+the napari viewer.
+"""
+
+import napari
+from time import sleep
+from napari.qt.progress import progress
```

Comment:
pending the in-qt vs not-in-qt discussion, this should be put in `_progress.py` and then imported in `qt/__init__.py` so that you don't need the triple import: `from napari.qt import progress`